src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -471,6 +488,11 @@ bool BLEManagerImpl::SendIndication(BLE_CONNECTION_OBJECT conId, const ChipBleUU
     int32_t status = 0;
     status = rsi_ble_indicate_value(event_msg.resp_enh_conn.dev_addr, event_msg.rsi_ble_measurement_hndl, (data->DataLength()),
                                     data->Start());
+
+    // start timer for light indication confirmation. Long delay for spake2 indication
```

Comment:
I think there might be a typo in this comment.

Reply:
changed the comment